cbor/src/main/java/com/fasterxml/jackson/dataformat/cbor/CBORParser.java

```diff
@@ -1915,6 +1931,20 @@ private final byte[] _getBinaryFromString(Base64Variant variant) throws IOExcept
         return _binaryValue;
     }
 
+    /**
+     * Checking whether the current token represents an `undefined` value (0xF7).
+     * <p>
+     * This method allows distinguishing between real {@code null} and `undefined`,
+     * even if {@link CBORParser.Feature#HANDLE_UNDEFINED_AS_EMBEDDED_OBJECT} is disabled
+     * and the token is reported as {@link JsonToken#VALUE_NULL}.
+     *
+     * @return {@code true} if current token is an `undefined`, {@code false} otherwise
+     * @since 2.20
+     */
+    public boolean isUndefined() {
+        return (_inputBuffer[_inputPtr - 1] & 0xFF) == SIMPLE_VALUE_UNDEFINED;
```

Comment:
Hmmmh. Not a fan of this approach but... I guess it'll do. Let's preface it by

    (_currToken == JsonToken.VALUE_EMBEDDED_OBJECT || _currToken == JsonToken.VALUE_NULL)

to avoid likelihood of incorrect deterimation (binary segment ending with byte `0xF7`).

Also: not 100% sure if bounds checks would be needed.
Actually, must check that `_inputBuffer` is not `null` -- `_releaseBuffers()` clears it.

Reply:
Actually I can add that.